elastic/kibana
# [Lens] Handle failing existence check

Part of https://github.com/elastic/kibana/issues/70520

![existence_error_handling](https://user-images.githubusercontent.com/1508364/86476728-5e430680-bd47-11ea-8672-8107577da700.gif)

Currently if the field existence check of the data panel fails, the loading spinner will be shown forever. This PR handles it gracefully:
* List all fields from the index pattern saved object as available
* Show a warning icon the availability request didn't work

To test: 
* Install two index patterns
* Go to Lens
* Switch between both to make sure the local index pattern cache is poplulated
* Set network to offline in dev tools
* Switch index pattern again
